hack/machineset.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
+
+# machineset.sh - create a Windows MachineSet for Windows Machine Config Operator
```

Comment:
Please update the MachineSet section of the README with help regarding this script.

Reply:
Forgot about this part... adding while addressing next wave of changes